py3status/modules/getjson.py

```diff
@@ -68,28 +65,17 @@ class Py3status:
     def getjson(self):
         """
         """
-        response = {
-            'cached_until': self.py3.time_in(self.cache_timeout),
-        }
-
         try:
-            resp = self.py3.request(self.url, timeout=self.timeout)
-            status = resp.status_code == 200
-            resp = resp.json()
+            getjson = self.py3.request(self.url, timeout=self.timeout).json()
+            getjson = self.py3.flatten_dict(
+                getjson, delimiter=self.delimiter, intermediates=True)
         except self.py3.RequestException:
-            resp = None
-            status = False
-
-        if status:
-            response['full_text'] = self.py3.safe_format(
-                self.format,
-                self.py3.flatten_dict(
-                    resp, delimiter=self.delimiter, intermediates=True
-                )
-            )
-        else:
-            response['full_text'] = ''
-        return response
+            getjson = None
```

Comment:
I think we should use `[{}]`. That's what the formatter is for. When I was fixing `gpmdp`, I had wrong placeholders in. I forget we need to use `{song_artist}` instead of `{artist}` and `{song_title}` instead of `{title}`.

Hiding placeholders right away could make easy fixes turning into a much harder fixes for everybody. We get users coming in, saying they tried something... and it didn't work... Oh it's because they put in wrong placeholders, deprecated placeholders, or typos in placeholders, etc.

P.S. Being a detective suck... Hunting down bugs with tools that does not work as expected because things get blocked, things get hidden, etc... It's not a bug,  it's a feature, they all said. 🤕